modules/ROOT/pages/runtime-patch-updates.adoc

```diff
@@ -0,0 +1,55 @@
+= Mule Runtime Patch Updates for Runtime Fabric 
+
+The following provides important information about Mule runtime engine patch updates for Runtime Fabric.
+
+== Monthly Date-Patch Updates 
+
+Once per month, MuleSoft releases a _date-patch update_,  which includes backward-compatible bug fixes to address any issues discovered in Mule runtime engine.
+The monthly date-patch update might include security patches if the scheduled date is within the security SLA.
+
+The format of the version number for these updates is:
+
+_Major_._Minor_._Patch_ - _Date_ 
+
+Here's an example version number, including the patch date:
+
+`4.3.0:20210101`
+
+The monthly patches update applications only to the latest date patch for the patch version.
+They do not change the patch version number.
```

Comment:
This is somewhat confusing. Perhaps we could say "They do not change the Runtime Version" instead?

Reply:
Ok, fixed.